src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -180,10 +180,29 @@ public boolean atDesiredPosition() {
   }
 
   /**
-   * @return true if the elevator's current position is less than its interference threshold
+   * @return true if the elevator's current position is less than the elevator-intake interference
+   *     zone
    */
-  public boolean elevatorBelowInterferenceThreshold() {
-    return leftMotorEncoderRel.getPosition() < ElevatorCal.ELEVATOR_INTERFERENCE_THRESHOLD_INCHES;
+  public boolean elevatorBelowInterferenceZone() {
+    return leftMotorEncoderRel.getPosition()
+        < ElevatorCal.ELEVATOR_INTERFERENCE_THRESHOLD_MINIMUM_INCHES;
+  }
+
+  /**
+   * @return true if the elevator's current position is greater than the elevator-intake
+   *     interference zone
+   */
+  public boolean elevatorAboveInterferenceZone() {
+    return leftMotorEncoderRel.getPosition()
+        > ElevatorCal.ELEVATOR_INTERFERENCE_THRESHOLD_MAXIMUM_INCHES;
+  }
+
+  /**
+   * @return true if the elevator's current position is outside the elevator-intake interference
+   *     zone
+   */
+  public boolean elevatorOutsideInterferenceZone() {
+    return elevatorBelowInterferenceZone() || elevatorAboveInterferenceZone();
```

Comment:
I can't think of any use cases for this

Reply:
Well I was using it but then that was wrong, so now I've deleted it